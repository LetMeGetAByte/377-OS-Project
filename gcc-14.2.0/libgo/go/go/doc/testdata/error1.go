version https://git-lfs.github.com/spec/v1
oid sha256:6fca67868db1c7af9e9cff92aa12135968d789cf011fec8c4916c92aed65341c
size 498
