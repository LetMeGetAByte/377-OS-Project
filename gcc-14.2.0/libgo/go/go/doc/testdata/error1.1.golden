version https://git-lfs.github.com/spec/v1
oid sha256:0ba37dd88bc0a9de738394eb2c1e9be56e5f8f4b1e0180b18236d4e0709b633c
size 440
